guides/v2.2/get-started/soap/soap-web-api-calls.md

```diff
@@ -17,33 +17,10 @@ Each Magento service interface that is part of a [service contract](https://glos
 
 To consume several services, you must specify them in the WSDL endpoint [URL](https://glossary.magento.com/url).
 
-
-<table style="width:100%">
-   <colgroup>
-      <col width="20%" />
-      <col width="40%" />
-      <col width="40%" />
-   </colgroup>
-   <thead>
-      <tr>
-         <th>Service</th>
-         <th>WSDL endpoint URL</th>
-         <th>Available services</th>
-      </tr>
-   </thead>
-   <tbody>
-      <tr>
-         <td>customer</td>
-         <td>http://magentohost/soap?wsdl&services=customerV1</td>
-         <td>\Magento\Customer\Service\V1\CustomerService</td>
-      </tr>
-       <tr>
-         <td>customer, catalogProduct</td>
-         <td>http://magentohost/soap/custom_store?wsdl&services=customerCustomerAccountServiceV1,catalogProductV2</td>
-         <td>\Magento\Customer\Service\V1\CustomerAccountServiceInterface, \Magento\Catalog\Service\V2\ProductInterface</td>
-      </tr>
-   </tbody>
-</table>
+| Service | WSDL endpoint URL | Available services |
+| --------- | ---------- | ------------------------------------------ |
+| customer | http://magentohost/soap?wsdl&services=customerV1 | \Magento\Customer\Service\V1\CustomerService |
+| customer, catalogProduct | http://magentohost/soap/custom_store?wsdl&services=customerCustomerAccountServiceV1,catalogProductV2 | \Magento\Customer\Service\V1\CustomerAccountServiceInterface, \Magento\Catalog\Service\V2\ProductInterface |
```

Comment:
Is this correct? Do we have any V2s?

Reply:
Looks like it's not. Currently there are only V1 services available. Also looks like the `Magento\Customer\Service\V1\CustomerAccountServiceInterface` and `Magento\Customer\Service\V1\CustomerAccountService` classes are obsolete, so they were also adjusted with the corresponding API classes.

Thanks.